docs/source/configuration.ipynb

```diff
@@ -126,7 +126,52 @@
     "      [Enter MyProxy pass phrase]\n",
     "      [you should receive a credential in /tmp/x509up_u1000]\n",
     "      gsissh -p 2222 login1.stampede2.tacc.utexas.edu\n",
-    "      ```\n"
+    "      ```\n",
+    "\n",
+    "- `interactive` - RP can operate from within a set of compute nodes interactivley.\n",
+    "\n",
+    "   - Specify the `pd.access_schema = interactive`\n",
+    "   - Request compute nodes interactively on any resource. Here we are requesting 2 nodes\n",
+    "      for 1 hour:\n",
+    "\n",
+    "      `$ srun --pty -A xxxx -N 2 -n 56 -t 01:00:00 -p development /bin/bash -l`\n",
+    "   \n",
+    "\n",
+    "   - Export `RADICAL_PILOT_DBURL` path:\n",
+    "\n",
+    "      `$ export RADICAL_PILOT_DBURL=mongodb://user:password@ip/db_name`\n",
+    "      \n",
+    "   - Run RP within these requested resources.\n",
+    "\n",
+    "      `$ python 00_getting_started.py`\n",
+    "   \n",
+    "- `batch_mode`: The native way of running applications on HPC is by creating a [batch_job](https://hpc-wiki.info/hpc/FAQ_Batch_Jobs) script describing the resource requirements for your application.\n",
+    "     For example, to run RP as a batch_job:\n",
+    "\n",
+    "   - Specify the `pd.access_schema = interactive`\n",
```

Comment:
`pd.access_schema = 'batch_mode'`

btw, maybe it should be just `batch`? (kinda we don't set `interactive_mode`)

Reply:
I agree with that.

Reply:
I agree as well.